lib/Common/Memory/ArenaAllocator.h

```diff
@@ -801,17 +801,16 @@ class RefCounted
 {
     volatile LONG refCount;
 
-protected:
-    virtual ~RefCounted()
-    {
-    }
-
 public:
     RefCounted()
         : refCount(1)
     {
     }
 
+    virtual ~RefCounted()
```

Comment:
I think the intent here is that RefCounted should never be instantiated by itself.
May be it should put the constructor as protected instead.